ls1intum/Artemis
# Bugfix/Programming-Exercises/Sequential test runs flag not serialized in JSON

### Checklist

### Motivation and Context



### Description


The sequential test run flag is not serialized into JSON as the JsonProperty annotation was missing.
(This would also work appearently if the getter would be named `getSequentialTestRuns` but I like it better as `hasSequentialTestRuns`)

### Steps for Testing


1. Log in to ArTEMiS
2. Open a programming exercise in course administration > view
3. Sequential test run flag should show in screen

### Screenshots



Before:

![image](https://user-images.githubusercontent.com/28230611/62870876-ac5baf80-bd1a-11e9-9692-598c6a99d85a.png)

Now:

![image](https://user-images.githubusercontent.com/28230611/62870847-9c43d000-bd1a-11e9-970f-e1279198682e.png)
